go.mod

```diff
@@ -3,41 +3,41 @@ module github.com/influxdata/influxdb/v2
 go 1.13
 
 require (
+	cloud.google.com/go/bigtable v1.3.0 // indirect
```

Comment:
We should not change anything in go.mod/go.sum as part of this PR.

Reply:
@jacobmarble I'm not sure how that would be possible, as we're moving away form the goreleaser fork to the offficial release